README.md

```diff
@@ -18,11 +29,14 @@ We've excluded the number emojis due to their lack of support on Chrome. We enco
 Just to clarify - it is **not required** to run gulp for the meetup. It's only being used to compile the styles. However if you'd like to play around on your own time, follow the steps below to get setup.
 
 #### 1. Install Dependencies
-First, ensure you have `Node.JS` installed. Download that [here](https://nodejs.org/en/).
+First, ensure you have Node.js installed. Download that [here](https://nodejs.org/en/).
```

Comment:
This was lowercased for consistency's sake (see line 16). We can 💯  change it both to caps!